spec/support/schemas/session/destroy.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module DestroySessionSchema
```

Comment:
let's rename to `Session::DestroySchema`
also, let's rename a file to `destroy_schema.rb`